docs/.vitepress/theme/components/Layout.vue

```diff
@@ -170,7 +170,7 @@ const { frontmatter } = useData() as any
           @click="mobileMenuOpen = false"
         />
       </aside>
-      <main class="w-[800px] mx-[24px] xl:mx-auto md:mt-[24px]">
+      <main class="w-[800px] mx-[24px] md:mt-[24px]">
```

Comment:
How about this? this creates a custom breakpoint exactly where the problem would be. Check it out!

```suggestion
      <main class="w-[800px] mx-[24px] min-[1420px]:mx-auto md:mt-[24px]">
```